source/Magritte-Model.package/Object.extension/instance/maLazyFromDescriptionOf..st

```diff
@@ -0,0 +1,20 @@
+*Magritte-Model
+maLazyFromDescriptionOf: accessorSymbol
+	"Return the current value of a field, or the default specified in the description
+
+	Usage: 
+		MyDomainObject>>#getter
+			^ self maLazyFromDescriptionOf: #getter
+		
+		MyDomainObject>>#getterDescription
+			^ MAElementDescription new
+					accessor: #getter;
+					default: 1;
+					yourself"
+	| currentValue defaultValue description |
+	currentValue := self instVarNamed: accessorSymbol.
+	currentValue ifNotNil: [ ^ currentValue ].
+	description := self magritteDescription children detect: [ :desc | desc isMessage: accessorSymbol ].
+	defaultValue := description default.
```

Comment:
How is supposed to work with values that should be nil? I case of the getter read accessor you do not return nil if this is the value but changing the object and returning default value. This can introduce a lot of subtle bugs